model/src/main/kotlin/io/spine/validation/OptionExts.kt

```diff
@@ -31,18 +31,17 @@ package io.spine.validation
 import com.google.protobuf.BoolValue
 import com.google.protobuf.GeneratedMessage.GeneratedExtension
 import io.spine.protobuf.unpack
+import io.spine.protodata.ast.Field
 import io.spine.protodata.ast.Option
 
 /**
- * Checks if this option represents the given generated option.
+ * Finds the option with the given generated type applied to this [Field].
```

Comment:
Maybe it's "generated extension type"?  I mean all the types generated out of Protobuf definitions are generated. Here the parameter has a precise type.